src/Sylius/Bundle/ApiBundle/composer.json

```diff
@@ -43,7 +43,6 @@
         "symfony/browser-kit": "^5.4.21 || ^6.0",
         "symfony/debug-bundle": "^5.4.21 || ^6.0",
         "symfony/dotenv": "^5.4.21 || ^6.0",
-        "symfony/http-client": "^5.4.21 || ^6.0",
```

Comment:
If nothing changed in the API bundle, why is this not a dev requirement anymore? Was it here by mistake?

Reply:
I asked that same question on our daily :D. As it's required by Core component it's no longer required to be included in the ApiBundle composer.json, as it's installed anyway (ApiBundle required CoreBundle which requires Core component).